index.html

```diff
@@ -13,7 +13,7 @@
 <body>
     <div class="container">
         <div class="card1">
-            <img src="image/image2.png" alt="">
+            <img src="image/image2.png" alt="hello">
         <h1>TONIC</h1> 
           <ul>
            <li><h3>CANOPY</h3></li>
```

Comment:
- [ ] Please you should use only one h1 element in the document since this will improve the SEO.